src/components/search/SearchSection.jsx

```diff
@@ -6,6 +6,4 @@ const SearchSection = () => {
   );
 };
 
-
-
-export default SearchSection;
\ No newline at end of file
+export default SearchSection;
```

Comment:
Added a newline in accordance with the [eol-last](https://eslint.org/docs/rules/eol-last) rule.